doc/design/paths-endpoint-webdav.md

```diff
@@ -0,0 +1,21 @@
+# Discussion on metadata needed for dandidav
+
+Submitted as a PR to avoid spagetti thread discussion on https://github.com/dandi/dandi-archive/issues/1837#issuecomment-1921864949
+
+
+@jwodder listed needed properties and metadata fields:
+
+* Asset properties used by dandidav:
+    * `asset_id`
```

Comment:
```suggestion
    * `asset_id` (PRESENT)
```

But I guess only to get metadata, and should generally not be needed if contentURL to redirect to is known, right?

Reply:
It's needed for both metadata and properties, as the only properties that `../assets/paths/` provides are path and asset ID.

Reply:
and `aggregate_size` which is the size for the asset if I got it right